cluster-service/deploy/templates/azure-operators-managed-identities-config.configmap.yaml

```diff
@@ -8,52 +8,102 @@ data:
     controlPlaneOperatorsIdentities:
       cluster-api-azure:
         minOpenShiftVersion: 4.18
+        roleDefinitions:
```

Comment:
Assuming that this defines a roleDefinitions where the value is an array of structs where the structs have the resourceId and name keys it seems good to me

Reply:
Yes indeed it is an array of roledefinitions.. I had to deploy a dev env .. and I deployed with this configuration .. have a look at the CS configmap:
```
file-csi-driver:
        minOpenShiftVersion: 4.18
        roleDefinitions:
          - resourceId: '/providers/Microsoft.Authorization/roleDefinitions/xxxxxxxxxxxxxx'
            name: 'File Storage Operator - Dev'
        azureRoleDefinitionResourceId: '/providers/Microsoft.Authorization/roleDefinitions/xxxxxxxxxxxxxxxxxx'
        azureRoleDefinitionName: 'File Storage Operator - Dev'
```